app/services/caesar_importer.rb

```diff
@@ -0,0 +1,77 @@
+class CaesarImporter
+  attr_reader :reduction_id, :reducible, :data, :subject_info
+
+  def initialize(**args)
+    @reduction_id = args[:id]
+    @reducible = args[:reducible]
+    @data = args[:data]
+    @subject_info = args[:subject]
+  end
+
+  def process
+    generate_parent_resources
+
+    # If a transcription with this id already exists, an exception will be raised
+    new_transcription = Transcription.new(transcription_create_attrs)
+    new_transcription.workflow = workflow
+    new_transcription.save!
+  end
+
+  private
+
+  def workflow
+    Workflow.find_by(id: reducible[:id])
```

Comment:
So this is probably just a matter of me not being in the loop about how caesar data is structured exactly, _but_, I figured I'd ask anyways-
This `reducible` that we're receiving from Caesar, is its `id` the `workflow id` by design?

Reply:
Addressed below